extensions/amp-analytics/0.1/variables.js

```diff
@@ -213,10 +214,20 @@ export class VariableService {
   }
 
   /**
+   * @param {Element=} element
    * @return {!JsonObject} contains all registered macros
    */
-  getMacros() {
-    return this.macros_;
+  getMacros(element) {
```

Comment:
When would the element be undefined?

Reply:
Yeah originally I was thinking we may not want every caller to have access to these macros, but now I think that they probably should. Went through all callers and made sure the element was passed in, and removed this.

Reply:
Thank you